Makefile

```diff
@@ -77,8 +77,7 @@ check:
 	go vet
 
 gosec:
-	gosec -quiet -log gosec.log -out=gosecresults.csv -fmt=csv ./...
-
+	go list ./... | grep -vE 'powerflexclient|apexclient|jobsclient' | xargs gosec -stdout -log gosec.log -out=gosecresults.csv -fmt=csv
```

Comment:
do you need to add the powerscaleclient here also?

Reply:
@doriac11 Good point, it didn't have any issues right now but good to add here.